app/src/utils/forms.py

```diff
@@ -0,0 +1,9 @@
+# -*- coding: utf-8 -*-
+from django.utils.safestring import mark_safe
+
+
+def errors_format(form):
+    infos = ["<span><strong>" + field + "</strong><br/>" + "<br/>".join(errs) + "</span>"
+             for field, errs in form.errors.items()]
+
+    return mark_safe("<p>" + "<br/><br/>".join(infos) + "</p>")
```

Comment:
Is it possible that `infos` will contains **any** output that originally will comes from users? If that's true you need to escape characters in that string first.

Reply:
It depends on what you put into errors. I think it should contain no user input, but additional escaping won't hurt ;)